tools/downloader/common.py

```diff
@@ -228,6 +228,7 @@ def apply(self, output_dir):
         print('========= Unpacking {} ========='.format(postproc_file))
 
         shutil.unpack_archive(str(postproc_file), str(output_dir), self.format)
+        postproc_file.unlink()  # Remove an archive
```

Comment:
This isn't a bad idea, but don't sneak it into an unrelated PR. Submit it separately.

Reply:
Opened https://github.com/opencv/open_model_zoo/pull/376